packages/react-tooltip/src/stories/TooltipStyled.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { Button } from '@fluentui/react-button';
+import { SlideTextRegular } from '@fluentui/react-icons';
+import { makeStyles } from '@griffel/react';
+
+const getStyles = makeStyles({
+  toolTip: {
+    backgroundColor: 'green',
+    color: 'white',
```

Comment:
Can we use `tokens` instead to ensure that customers use good patterns based on examples?

Reply:
i debated on tokens or not, my goal was to keep the demo incredibly generic, and not introduce more imports and concepts. Was trying to think of a style someone might do that didn't use shorthands, tokens, or anything like that.

Reply:
Pulled in tokens and it doesn't feel like it jumped in complexity too much. I'm good with it